lib/chemistry_rails/orm/activerecord.rb

```diff
@@ -7,7 +7,7 @@ module ActiveRecord
 
     def chemical_formula(column)
 
-      validates_format_of column, with: /\A((#{ChemistryRails::ELEMENTS.reject(&:nil?).map{|i| i[:short] }.join('|')})+[0-9]*)+\Z/, message: :chemistry_rails_formula
+      validates_format_of column, with: /\A(((#{ChemistryRails::ELEMENTS.reject(&:nil?).map{|i| i[:short] }.join('|')})+[0-9]*)+|(\s)*)\Z/, message: :chemistry_rails_formula
```

Comment:
I think it will be better to do something like this:
validates_format_of column, with: /....../, if "#{column}.present?"